langfuse.tf

```diff
@@ -26,8 +26,8 @@ clickhouse:
     existingSecretKey: clickhouse-password
 redis:
   deploy: false
-  host: ${azurerm_redis_cache.this.name}.redis.cache.windows.net
-  port: 6380
+  host: ${local.globally_unique_prefix}${var.name}.${var.location}.${var.use_redis_enterprise ? "redisenterprise" : "redis"}.cache.azure.net
```

Comment:
Even though the Azure UI shows this resulting hostname, the DNS resolution within the langfuse-web container fails. So this will need further investigation.

I tried to use the Private Link DNS FQDN here as well, but this shows a certificate resolution problem since westeurope.privatelink.redisenterprise.cache.azure.net is not part of the certificate.